lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -67,7 +67,6 @@
                     // name is being routed to.
                     router = this.router = new Backbone.Router();
                     _.each([
-                        [':default', _.bind(this.routeNotFound, this)],
```

Comment:
I removed this because it was showing the "page #main not found" warning when the #main skip link was used. Unsure what else this implicates though. @andy-armstrong?